### PR TITLE
Add `sort-direction` option

### PR DESCRIPTION
**Pull request by TimonVS**
This PR adds the `sort-direction` option. Valid values are `ascending` or `descending`. I do notice that the list of configuration options is getting quite long, we may want to split up more advanced configuration options like this one into another document in the near future.

Fixes: #188

### CODE REVIEW

**Comment by TimonVS** [comment]
I can't select people who aren't a member of the repo as a reviewer 🤔. But I'd appreciate @casz review as well 🙇‍♂️.

**Comment by casz** [comment]
I'd be happy to be considered a member of the repo :+1: :muscle:

File: index.js [hunk 1]
```diff
@@ -27,6 +33,7 @@ module.exports = app => {
       context,
       replacers: config.replacers
     })
+    config['sort-direction'] = validateSortDirection(config['sort-direction'])
```
Comment (TimonVS) [inline index.js]: It's probably better to solve this with a Joi schema?

Comment (casz) [reply index.js]: I'll see if I can get around to it today :+1:

Comment (TimonVS) [reply index.js]: No need to hurry! If you don't get around to it we can simply create an issue to improve this later :)

File: lib/sort-pull-requests.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const SORT_DIRECTIONS = {
+  ascending: 'ascending',
+  descending: 'descending'
+}
+
+module.exports.SORT_DIRECTIONS = SORT_DIRECTIONS
+
+module.exports.validateSortDirection = sortDirection => {
+  console.log(Object.keys(SORT_DIRECTIONS).includes(sortDirection))
```
Comment (toolmantim) [inline lib/sort-pull-requests.js]: Did you mean to leave this in?

**Comment by casz** [comment]
Should be easy for me to add into schema so perhaps my PR supersedes his? :laughing:

**Comment by toolmantim** [comment]
So, after doing a quick test, I'm thinking we should reverse the default sort order @TimonVS, and make it `ascending`. What do you reckon from your testing?

**Comment by TimonVS** [comment]
Your call, `ascending` kind of was the default before so I'm fine with changing it to `ascending`. Personally I think I prefer `descending` but I don't have a strong opinion on it :)

**Comment by toolmantim** [comment]
Cool. I remember why I thought that maybe `ascending` was better again — I found while editing the latest release notes for Release Drafter, if you have PRs that build upon one another, they don't make much sense in reverse chronological order and takes more mental gymnastics.

But TBH, I'm not really fussed enough to change it either. Let's wait for feedback and see.

**Comment by casz** [comment]
My 2 cents is I found ascending confusing, took me a while to get over 😆 however I much prefer descending.

**Pull request merged**